### PR TITLE
Scala 213 support

### PR DESCRIPTION
**Pull request by sriraamas**

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -132,3 +132,12 @@ load("@io_bazel_rules_play_routes//:workspace.bzl", "play_routes_repositories")
 play_routes_repositories(play_version)
 load("@play_routes//:defs.bzl", play_routes_pinned_maven_install = "pinned_maven_install")
 play_routes_pinned_maven_install()
+
+load("@bazel_tools//tools/build_defs/repo:jvm.bzl", "jvm_maven_import_external")
+
+jvm_maven_import_external(
```
Comment (SrodriguezO) [inline WORKSPACE]: Can we import this with rules_jvm_external instead?

File: test_workspace.bzl [hunk 2]
```diff
@@ -8,13 +8,13 @@ def play_routes_compiler_cli_test_repositories():
...
-            "org.specs2:specs2-matcher_2.12:4.7.0",
-            "com.typesafe.akka:akka-actor_2.12:2.5.24",
-            "com.typesafe.play:play_2.12:2.7.4",
-            "com.typesafe.play:play-test_2.12:2.7.4",
-            "com.typesafe.play:play-specs2_2.12:2.7.4",
+            "org.specs2:specs2-common_2.13:4.7.0",
+            "org.specs2:specs2-core_2.13:4.7.0",
+            "org.specs2:specs2-matcher_2.13:4.7.0",
+            "com.typesafe.akka:akka-actor_2.13:2.5.24",
+            "com.typesafe.play:play_2.13:2.7.4",
```
Comment (SrodriguezO) [inline test_workspace.bzl]: should these use the play 2.8.2 versions?

**Review by SrodriguezO — CHANGES_REQUESTED** [review]
Looks good to me other than the small comments I left :)

File: workspace.bzl [hunk 3]
```diff
@@ -8,14 +8,14 @@ def play_routes_compiler_cli_repositories():
     maven_install(
         name = "play_routes_compiler_cli_maven",
         artifacts = [
-            "com.github.scopt:scopt_2.12:3.7.0",
-            "com.typesafe.play:routes-compiler_2.12:2.7.4",
-            "org.scala-sbt:zinc_2.12:1.2.1",
+            "com.github.scopt:scopt_2.13:3.7.1",
+            "com.typesafe.play:routes-compiler_2.13:2.8.2",
+            "org.scala-sbt:zinc_2.13:1.4.0-M6",
```
Comment (SrodriguezO) [inline workspace.bzl]: Hmm this is a milestone release; are we okay with using that? It might be less stable

File: WORKSPACE (hunk 1 above)
Comment (sriraamas) [reply WORKSPACE]: I don't think it worked, but I will try again.

File: test_workspace.bzl (hunk 2 above)
Comment (sriraamas) [reply test_workspace.bzl]: ya. I will update the test ones.

**Review by SrodriguezO — APPROVED** [review]
lgtm; thanks!

**Pull request merged**